modules/virt-creating-secondary-localnet-udn.adoc

```diff
@@ -0,0 +1,85 @@
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: '' # <2>
+  desiredState:
+    ovn:
+      bridge-mappings:
+      - localnet: tenantblue # <3>
+        bridge: br-ex # <4>
+        state: present # <5>
+----
+<1> The name of the configuration object.
+<2> Specifies the nodes to which the node network configuration policy is to be applied. The recommended node selector value is `node-role.kubernetes.io/worker: ''`.
+<3> The name of the additional network from which traffic is forwarded to the OVS bridge. This attribute must match the value of the `spec.config.name` field of the `NetworkAttachmentDefinition` object that defines the OVN-Kubernetes additional network.
+<4> The name of the OVS bridge on the node. This value is required if the `state` attribute is `present` or not specified.
+<5> The state of the mapping. Must be either `present` to add the mapping or `absent` to remove the mapping. The default value is `present`.
++
+[NOTE]
+====
+{VirtProductName} does not support Linux bridge bonding modes 0, 5, and 6. For more information, see link:https://access.redhat.com/solutions/67546[Which bonding modes work when used with a bridge that virtual machine guests or containers connect to?].
+====
+
+. Create a `ClusterUserDefinedNetwork` object to create a localnet secondary network:
++
+[source,yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: ClusterUserDefinedNetwork
+metadata:
+  name: cudn-localnet # <1>
+spec:
+  namespaceSelector: # <2>
+    matchExpressions: # <3>
+    - key: kubernetes.io/metadata.name
+      operator: In # <4>
+      values: ["red", "blue"]
+  network:
+    topology: Localnet # <5>
+    localnet:
+        role: Secondary # <6>
+        physicalNetworkName: tenantblue # <7>
+        ipam:
+          mode: Disabled # <8>
+# ...
+----
+<1> The name of the `ClusterUserDefinedNetwork` custom resource.
+<2> The set of namespaces that the cluster UDN applies to. The namespace selector must not point to `default`, an `openshift-*` namespace, or any global namespaces that are defined by the Cluster Network Operator (CNO).
+<3> The type of selector. In this example, the `matchExpressions` selector selects objects that have the label `kubernetes.io/metadata.name` with the value `red-namespace` or `blue-namespace`.
+<4> The type of operator. Possible values are `In`, `NotIn`, and `Exists`.
+<5> The topological configuration of the network. A `Localnet` topology connects the logical network to the physical underlay.
+<6> Specifies if the UDN is primary or secondary. The required value is `Secondary` for `topology: Localnet`.
+<7> The name of the OVN bridge mapping that is configured on the node. This value must match the `spec.desiredState.ovn.bridge-mappings.localnet` field in the `NodeNetworkConfigurationPolicy` manifest that you previously created. This ensures that you are bridging to the intended segment of your physical network.
+<8> Specifies that configuring IP address management (IPAM) for VMs is not supported.
```

Comment:
Is this a default value or it must be set to `Disabled` in this case?

Reply:
AFAIK, it must be set to `Disabled`